regression-test/suites/bloom_filter_p0/test_bloom_filter_hit.groovy

```diff
@@ -129,6 +129,7 @@ suite("test_bloom_filter_hit") {
     sql """ set parallel_scan_min_rows_per_scanner = 2097152; """
 
     // bf filter
+    sql """ SET parallel_fragment_exec_instance_num=1 """
```

Comment:
set parallel_pipeline_task_num=1 to make it work on pipeline engine, parallel_fragment_exec_instance_num is deprecated now